modules/claims_api/app/sidekiq/claims_api/va_notify_declined_job.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VANotifyDeclinedJob < ClaimsApi::ServiceBase
+    LOG_TAG = 'va_notify_declined_job'
+
+    def perform(encrypted_ptcpnt_id:, encrypted_first_name:, representative_id:)
+      lockbox = Lockbox.new(key: Settings.lockbox.master_key)
+      ptcpnt_id = lockbox.decrypt(encrypted_ptcpnt_id)
+      first_name = lockbox.decrypt(encrypted_first_name)
+      representative = ::Veteran::Service::Representative.find_by(representative_id:)
+
+      if representative.blank?
+        raise ClaimsApi::Common::Exceptions::Lighthouse::ResourceNotFound.new(
+          detail: "Could not find veteran representative with id: #{representative_id}"
+        )
+      end
+
+      res = send_declined_notification(ptcpnt_id:, first_name:, representative:)
+
+      ClaimsApi::VANotifyFollowUpJob.perform_async(res.id) if res.present?
+    rescue => e
+      msg = "VA Notify email notification failed to send with error #{e}"
+      slack_client = SlackNotify::Client.new(webhook_url: Settings.claims_api.slack.webhook_url,
+                                             channel: '#api-benefits-claims-alerts',
+                                             username: 'Failed ClaimsApi::VANotifyDeclinedJob')
+
+      slack_client.notify(msg)
```

Comment:
Could we utilize `slack_alert_on_failure` in `service_base` here?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/19362/commits/97ccbdb47037fe4fe8ce315c450a8c1b264bdbb2